test/functional/apps/settings/_initial_state.js

```diff
@@ -14,7 +14,10 @@ define(function (require) {
 
         return scenarioManager.reload('emptyKibana')
         .then(function () {
-          return settingsPage.navigateTo();
+          return settingsPage.navigateTo()
+          .then(function () {
+            return settingsPage.clickIndicesTab();
```

Comment:
Rather than nest this, can you just chain it?

```js
scenarioManager.reload('emptyKibana')
.then(() => settingsPage.navigateTo())
.then(() => settingsPage.clickIndicesTab())
```

just seems nicer than

```js
scenarioManager.reload('emptyKibana')
.then(() => settingsPage.navigateTo().then(() => settingsPage.clickIndicesTab()))
```